keras/losses.py

```diff
@@ -1885,6 +1891,12 @@ def _ragged_tensor_sparse_categorical_crossentropy(y_true,
       sparse_categorical_crossentropy, from_logits=from_logits, axis=axis)
   return _ragged_tensor_apply_loss(fn, y_true, y_pred, y_pred_extra_dim=True)
 
+def _check_ground_truth_range(y_true):
+  """ Ensures ground truth labels in binary_crossentropy and 
+      binary_focal_crossentropy are between 0.0 and 1.0.
+  """
+  if tf.math.reduce_max(y_true) > 1.0 or tf.math.reduce_min(y_true) < 0.0:
```

Comment:
You can't do value checks like this, because that will only work in eager mode, not in graph mode. In general value checks should be avoided because they add computation overhead. But if they're absolutely necessary, they should be implemented via a symbolic op, such as `assert_less` -- https://www.tensorflow.org/api_docs/python/tf/debugging/assert_less

In this case, it does not seem like a good idea to add this check because of the overhead. If we do it, we should restrict it to only run in eager execution (by checking https://www.tensorflow.org/api_docs/python/tf/executing_eagerly)

Reply:
Can we just drop the PR, since it's understood that binary losses are supposed to be for 0's and 1's. Using for values not in range [0,1] makes no sense.